src/index.js

```diff
@@ -1,36 +1,2 @@
 import './style.css';
```

Comment:
- [ ] Then on your index.js you can do things like these. Webpack automatically will bundle all of the JS modules into one, our `bundle.js`. I recommend you read a bit more about bundlers and what are they work, so you don't face any more issues like this in the future.

If you have been following my recommendations until now, you index.js should like something like this.

![image](https://user-images.githubusercontent.com/79114233/145663033-551039b8-b57f-450b-9126-00d855562a36.png)

And the tasks now will be displayed correctly when we run our scripts, either `npm run start` or `npm run build`.